code/modules/mob/living/living_defines.dm

```diff
@@ -86,6 +86,8 @@
 	var/frozen = null
 	/// Used for keeping track of previous sleeping value with admin freeze.
 	var/admin_prev_sleeping = 0
+	/// Whether or not animal is only spawned/used for unit-test purposes
+	var/unit_test_dummy = FALSE
```

Comment:
Reckon this can be gated behind `#ifdef CI_TESTING`? I dont want an extra var on all `/mob/living` just for this one test.

Reply:
That would be possible, yeah. Will do that unless there’s some better place for the variable, or to achieve this?

Reply:
Fixed, removed the variable.